.nvmrc

```diff
@@ -1 +1 @@
-v12.16.3
+v14.5.0
```

Comment:
Should this match the specific node version you’re pinning on the Docker images?

Reply:
I _don't think_ a patch will cause us grief 🤔 I will update just to be sure though. Thanks :)